docs/logs/getting-started/README.md

```diff
@@ -53,13 +53,48 @@ What does the above program do?
 
 The program uses the
 [`ILogger`](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger)
-API to log a formatted string with a severity level of Information. Click
-[here](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.loglevel)
-for more information on the different logs levels. Opentelemetry captures this
-and sends it to `ConsoleExporter`. `ConsoleExporter` simply displays it on the
-console.
+API to log a formatted string with a severity level of Information. OpenTelemetry
+captures this and sends it to the `ConsoleExporter` which displays logs on the console.
+
+## Configure Filtering
```

Comment:
Why is this needed in a "getting started doc"?

Reply:
Lets stick with the doc title - "Getting started in 5 minutes", so anything not required to get up and running, can be added elsewhere.